### ISSUE TITLE
A11y: no direct access to the primary content of the Web page

### ISSUE THREAD

**Issue by ChristofSimon** [grafana/grafana#82188]
**Steps to reproduce**:
New dashboard - go to the first active element (Add a new panel)

**Actual Result**:
You have to press the Tab key 15 times to get to the first active element.
![2 4 1_bypass blocks_new dashbord](https://github.com/grafana/grafana/assets/122528177/4bf8a162-03a5-42c8-be0a-32ef84f050f2)

**Expected Result**
The keyboard focus should be directly on the control "Add a new panel" (primary content of the Web page) after selecting the menu item "New dashboard".

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
2.4.1 Bypass Blocks: https://www.w3.org/TR/WCAG21/#bypass-blocks

**Environment**:
- Grafana version: 9.1.0
- Data source type & version:
- User OS & Browser: Windows 10, Chrome Version 107.0.5304.88
- Others:

**Comment by ckbedwell** [comment]
Hi @ChristofSimon! Thanks for reporting all of these accessibility issues. This one was solved with the release of Grafana 10.1.0 with this PR: https://github.com/grafana/grafana/pull/68065 so I'm going to close this issue.
